### DISCUSSION TITLE
add group when import documents

### DISCUSSION THREAD

**Discussion by dromeuf** [Cinnamon/kotaemon#536] [General]
Hi,
Unless I've missed it in the interface. Would it be possible to add assignment to one or more groups when importing and indexing documents ?
Thanks for your great work, Kind regards, David.

**Comment by taprosoft** [comment]
Currently no. You can only add or remove files from group after it has been uploaded. But we will consider this in the backlog.

**Comment by eugr** [comment]
That would be incredibly useful, especially if you add a lot of files in one batch.
